cdc/sink/buffer_sink.go

```diff
@@ -61,21 +61,15 @@ func newBufferSink(
 type runState struct {
 	batch [maxFlushBatchSize]flushMsg
 
-	metricFlushDuration   prometheus.Observer
-	metricEmitRowDuration prometheus.Observer
```

Comment:
Why do we need to delete these two items?

Reply:
It's removed from monitoring and no need to track these metrics. Most of the time, these 2 do not cost too much time, might have some spikes, but do not have significant information, so removed.